botocore/loaders.py

```diff
@@ -233,29 +307,27 @@ class Loader:
 
     FILE_LOADER_CLASS = JSONFileLoader
     # The included models in botocore/data/ that we ship with botocore.
-    BUILTIN_DATA_PATH = os.path.join(BOTOCORE_ROOT, 'data')
+    BUILTIN_DATA_PATH = BOTOCORE_ROOT.joinpath('data')
     # For convenience we automatically add ~/.aws/models to the data path.
-    CUSTOMER_DATA_PATH = os.path.join(
-        os.path.expanduser('~'), '.aws', 'models'
-    )
+    # this filepath can never be zipped and will always be on disk
+    CUSTOMER_DATA_PATH = pathlib.Path('~', '.aws', 'models').expanduser()
+
     BUILTIN_EXTRAS_TYPES = ['sdk']
 
     def __init__(
         self,
         extra_search_paths=None,
         file_loader=None,
-        cache=None,
```

Comment:
This is a breaking change and will cause positional args to map to different values. Is there a reason this needed to be removed?

Reply:
was going to comment that i didn't see it being used anywhere in the code so thought it made to remove. there is a private `_cache` variable that is created at initialization, but does not reference this arg at all. Nor does the `instance_cache` decorator. i'll add it back